semaphore/run_mock_server.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+set -e
+
+#===============================================================================
+# A standalone script for running Dotcom with mock data.
+# Shold work both locally and on CI.
+# Runs Dotcom on port 8082 and mocks the API on port 8080 with Wiremock
+# If run with ENABLE_RECORDING=true, it runs Wiremock with --record-mappings,
+# and will overwrite some of the JSON files in apps/site/test/{mappings|__files}
+# with new API responses from the endpoint specified in WIREMOCK_PROXY_URL
+#
+# Run it like
+# bash ./semaphore/run_mock_server.sh &
+# or else it'll immediately exit and shut down the server.
+#===============================================================================
+
+# When script exits, clean up processes
+function cleanup () {
+  error_code=$?
+  printf -- '\033[35m :( Killing running processes... \033[0m\n'
+  pkill -TERM -f "wiremock"
+  pkill -TERM -f "mix phx.server"
+  pkill -TERM -f "sleep"
+  printf -- '\033[35m DONE.\nExiting....\n\n \033[0m\n'
+  exit "${error_code}";
+}
+trap cleanup ERR EXIT SIGINT SIGTERM
+
+#===============================================================================
+# Setting up Wiremock
+#===============================================================================
+
+export WIREMOCK_PROXY=true
+export WIREMOCK_PROXY_URL="${WIREMOCK_PROXY_URL:=https://dev.api.mbtace.com}"
```

Comment:
at some point, we were trying to keep the dev URLs out of the code... it may make sense to instead of defaulting, check for the existence of the variable and exit with a message if it is not present